plugin/build_baron.go

```diff
@@ -47,45 +50,78 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 		}
 		webhookConfigured := webHook.Endpoint != ""
 		if !webhookConfigured && proj.TicketCreateProject == "" {
-			return fmt.Errorf("ticket_create_project and taskAnnotationSettings.FileTicketWebHook endpoint cannot both be blank")
+			grip.Error(message.WrapError(err, message.Fields{
+				"message":      "ticket_create_project and taskAnnotationSettings.FileTicketWebHook endpoint cannot both be blank",
+				"project_name": projName,
+			}))
+			hasValidationError = true
 		}
 		if !webhookConfigured && len(proj.TicketSearchProjects) == 0 {
```

Comment:
Are we sure that there aren't places where we rely on this not being empty (because until now, this would always be > 0 or else the app wouldn't start) ? If yes, we may need to add some extra checks. Same with the other checks that we are removing. Now that we know longer have these guarantees, so we need additional checks anywhere?

Reply:
Same comment as [here](https://github.com/evergreen-ci/evergreen/pull/5021#discussion_r710201201), I think this should now be implemented in https://jira.mongodb.org/browse/EVG-15419